com.simplenote.Simplenote.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.simplenote.Simplenote</id>
+  <launchable type="desktop-id">com.simplenote.Simplenote.desktop</launchable>
+  <name>Simplenote</name>
+  <summary>Simplenote Desktop App</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <developer_name>Automattic Inc.</developer_name>
+  <url type="homepage">https://automattic.com/</url>
+  <description>
+    <p>
+      The simplest way to keep notes
```

Comment:
Looks like website has a little bit more info. Can you expand it?

Reply:
Additional information has been added.